### ISSUE TITLE
Divine Receptor can't be placed on the same item including Soul/Factor Receptor in affix process

### ISSUE THREAD

**Issue by velocity7** [malulleybovo/PSO2AffixingAssistant#27]
**Describe the bug**
See title

**To Reproduce**
Affix something such as Astral Soul, Ether Factor, Divine Order, and then attempt to have one of the fodder items be Soul Receptor, Factor Receptor, and Divine Receptor.

**Expected behavior**
Divine Receptor should be movable to the fodder item that includes Soul and Factor Receptor.

**Screenshots**
This fodder item should be transplantable according to PSO2, using a Divine Receptor capsule:
![image](https://user-images.githubusercontent.com/11874183/86378674-e9dd6a00-bc57-11ea-8135-829cf82de126.png)

**Comment by malulleybovo** [comment]
This issue has been resolved in the latest version.

**Comment by velocity7** [comment]
The issue still persists, I can't move Divine Receptor to a unit that has Soul Receptor and Factor Receptor:

![image](https://user-images.githubusercontent.com/11874183/86543697-fcbd9c00-beee-11ea-9c7a-483583ea82d5.png)

**Comment by malulleybovo** [comment]
Are you using version 1.1.5 of the Affixing Assistant? Look at the bottom left corner of the screen and check.
I replicated your formula (Astral Soul, Ether Factor, Divine Order, and Mana Reverie) but could not reproduce the unexpected result you shared. I got the expected result on v1.1.5:
![image](https://user-images.githubusercontent.com/14852459/86555629-85036780-bf16-11ea-85dd-a7af3f4ae419.png)
![image](https://user-images.githubusercontent.com/14852459/86555651-9482b080-bf16-11ea-9ba1-f29cf4692fbf.png)
![image](https://user-images.githubusercontent.com/14852459/86555694-b419d900-bf16-11ea-808f-a8c8084fd8f0.png)
